src/common/api/common/mail/FolderSystem.ts

```diff
@@ -174,6 +174,7 @@ const folderTypeToOrder: Record<SystemMailFolderTypes, number> = {
 	[MailSetKind.ARCHIVE]: 5,
 	[MailSetKind.SPAM]: 6,
 	[MailSetKind.ALL]: 7,
+	[MailSetKind.LABEL]: 8,
```

Comment:
Labels are not a folder type to order. In SystemMailFolderTypes, CUSTOM is excluded (since they are ordered a different way). LABEL should be excluded too.